src/abstractitensornetwork.jl

```diff
@@ -591,20 +591,22 @@ function neighbor_vertices(ψ::AbstractITensorNetwork, T::ITensor)
   return first.(v⃗)
 end
 
-function linkinds_combiners(tn::AbstractITensorNetwork)
+function linkinds_combiners(tn::AbstractITensorNetwork; edges_to_combine=edges(tn))
```

Comment:
I think calling it `edges` is fine, I don't think people will get confused about what that is referring to.